nipoppy_cli/nipoppy/workflows/dicom_reorg.py

```diff
@@ -13,7 +14,7 @@ class DicomReorgWorkflow(BaseWorkflow):
 
     def __init__(
         self,
-        dpath_root: Path | str,
+        dpath_root: StrOrPathLike,
         copy_files: bool = False,
         fpath_layout: Optional[Path] = None,
```

Comment:
in `pipeline.py` this is set as: `fpath_layout: Optional[StrOrPathLike] = None,` 
But several other places it's still `fpath_layout: Optional[Path] = None`

Reply:
Good catch, fixed!